### ISSUE TITLE
Families lacking an EURO SIGN glyph

### ISSUE THREAD

**Issue by felipesanches** [google/fonts#1229]
Below is a list (generated using FontBakery) of the 109 families from the Google Fonts collection that lack glyphs for EURO SIGN (unicode 0x22CA):

apache/droidarabickufi/DroidKufi
apache/droidarabicnaskh/DroidNaskh
apache/droidsans/DroidSans
apache/droidsansethiopic/DroidSansEthiopic
apache/droidsanstamil/DroidSansTamil
apache/droidsansthai/DroidSansThai
apache/droidserif/DroidSerif
apache/droidserifthai/DroidSerifThai
apache/jsmathcmbx10/jsMath
apache/jsmathcmex10/jsMath
apache/jsmathcmmi10/jsMath
apache/jsmathcmr10/jsMath
apache/jsmathcmsy10/jsMath
apache/jsmathcmti10/jsMath
apache/nokora/Nokora
ofl/adobeblank/AdobeBlank
ofl/aksarabaligalang/AksaraBaliGalang
ofl/amstelvaralpha/AmstelvarAlpha
ofl/angkor/Angkor
ofl/arefruqaa/ArefRuqaa
ofl/arvo/Arvo
ofl/astloch/Astloch
ofl/battambang/Battambang
ofl/bayon/Bayon
ofl/bentham/Bentham
ofl/bokor/Bokor
ofl/buda/Buda
ofl/chathura/Chathura
ofl/content/Content
ofl/decovaralpha/DecovarAlpha
ofl/dhurjati/Dhurjati
ofl/digitalnumbers/DigitalNumbers
ofl/enriqueta/Enriqueta
ofl/ewert/Ewert
ofl/farsan/Farsan
ofl/fasthand/Fasthand
ofl/geo/Geo
ofl/gurajada/Gurajada
ofl/hanna/BM
ofl/hannari/Hannari
ofl/hanuman/Hanuman
ofl/italiana/Italiana
ofl/jejugothic/JejuGothic
ofl/jejuhallasan/JejuHallasan
ofl/jejumyeongjo/JejuMyeongjo
ofl/josefinslab/JosefinSlab
ofl/kadwa/Kadwa
ofl/kantumruy/Kantumruy
ofl/karla/Karla
ofl/karlatamilinclined/KarlaTamilInclined
ofl/karlatamilupright/KarlaTamilUpright
ofl/kdamthmor/KdamThmor
ofl/kenia/Kenia
ofl/khula/Khula
ofl/khyay/Khyay
ofl/kokoro/Kokoro
ofl/kristi/Kristi
ofl/lakkireddy/LakkiReddy
ofl/lancelot/Lancelot
ofl/laomuangdon/LaoMuangDon
ofl/laomuangkhong/LaoMuangKhong
ofl/laosanspro/LaoSansPro
ofl/librebarcode39extended/LibreBarcode39Extended
ofl/librebarcode39extendedtext/LibreBarcode39ExtendedText
ofl/lohitbengali/Lohit
ofl/lohitdevanagari/Lohit
ofl/lohittamil/Lohit
ofl/mallanna/Mallanna
ofl/mandali/Mandali
ofl/meiescript/MeieScript
ofl/montserratsubrayada/MontserratSubrayada
ofl/myanmarsanspro/MyanmarSansPro
ofl/nicomoji/NicoMoji
ofl/nikukyu/Nikukyu
ofl/ntr/NTR
ofl/oxygen/Oxygen
ofl/padauk/Padauk
ofl/peddana/Peddana
ofl/phetsarath/Phetsarath
ofl/ponnala/Ponnala
ofl/puritan/Puritan
ofl/ramabhadra/Ramabhadra
ofl/ramaraja/Ramaraja
ofl/raviprakash/RaviPrakash
ofl/redactedscript/RedactedScript
ofl/revalia/Revalia
ofl/ruluko/Ruluko
ofl/sahitya/Sahitya
ofl/salsa/Salsa
ofl/sansita/Sansita
ofl/sarala/Sarala
ofl/sawarabigothic/SawarabiGothic
ofl/sawarabimincho/SawarabiMincho
ofl/seoulnamsanvertical/SeoulNamsanVertical
ofl/sitara/Sitara
ofl/souliyo/Souliyo
ofl/squadaone/SquadaOne
ofl/sreekrushnadevaraya/SreeKrushnadevaraya
ofl/suranna/Suranna
ofl/sura/Sura
ofl/suravaram/Suravaram
ofl/suwannaphum/Suwannaphum
ofl/tenaliramakrishna/TenaliRamakrishna
ofl/thabit/Thabit
ofl/timmana/Timmana
ofl/wendyone/WendyOne
ofl/yinmar/Yinmar
ofl/zillaslabhighlight/ZillaSlabHighlight
ofl/zillaslab/ZillaSlab

Interestingly, all fonts in the collection that provide the EURO SIGN glyph also do provide the EUROPEAN-CURRENCY SIGN glyph (codepoint 0x22A0). And all of the above families lack both glyphs.

**Comment by artyuum** [comment]
Agreed. I'm experiencing the same issue with "Karla" font. Euro sign (€) fallback to the default font because it does not seem to be supported by "Karla" font.
